### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,28 +3,69 @@ All notable changes to this project will be documented in this file.
...
---->
-###Android
+### Bugs
 
-####	 Updated
-
-* Base interactor not handling exceptions outside its domain anymore
+* BaseInteractor is not handling exceptions outside its domain anymore
 * List users with AssetListScreenlet
+* Fixed connection timeout in DDLFormScreenlet for upload files in hotfix/1.4.2
```
Comment (nhpatt) [inline CHANGELOG.md]: when uploading

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,28 +3,69 @@ All notable changes to this project will be documented in this file.
...
+* Propagated connection timeout in DDLFormScreenlet for upload files for Liferay 7.0
```
Comment (nhpatt) [inline CHANGELOG.md]: Propagating when uploading

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,28 +3,69 @@ All notable changes to this project will be documented in this file.
...
+* Changed wrong colors in buttons and dialogs
+* Used URIs instead of paths to support newer SDKs
+* Cache fails when storing blogs or uploading portraits
```
Comment (nhpatt) [inline CHANGELOG.md]: Fixed cache error

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,28 +3,69 @@ All notable changes to this project will be documented in this file.
...
+* Fixed corrupted credentials in LoginScreenlet stored in SharedPreferences
+* Fixed bug with camera and Android N
+* Changed minimun SDK because it has wrong target
```
Comment (nhpatt) [inline CHANGELOG.md]: Fixed **minimum** SDK in test application

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,28 +3,69 @@ All notable changes to this project will be documented in this file.
...
+
+### New features
+* Created connectors for support Liferay 6.2
```
Comment (nhpatt) [inline CHANGELOG.md]: to support

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,28 +3,69 @@ All notable changes to this project will be documented in this file.
...
+* Added Spanish translations
+* Prevent NullPointerException in login
+* Improvements to test-app
+* Added workaround to list Users with AssetListScreenlet
+* Improvements to PdfDisplayScreenlet: you can swipe between pages now
+* Exposed some methods to use in the POC of Nativescript
```
Comment (nhpatt) [inline CHANGELOG.md]: to use them

**Comment by victorg1991** [comment]
LGTM 👍

**Pull request merged**